javatests/google/registry/export/reporting_kinds.txt

```diff
@@ -0,0 +1,22 @@
+
```

Comment:
I'd omit the initial newline here.  I guess it shows one on backup_kinds.txt but that's actually an artifact of our export process that shouldn't really be there.